app/next-client-app/components/Tooltips.tsx

```diff
@@ -0,0 +1,26 @@
+import { Tooltip } from "react-tooltip";
+import { InfoIcon } from "lucide-react";
+
+export function Tooltips({
```

Comment:
Could we use the Shadcn component for this? https://ui.shadcn.com/docs/components/tooltip